Makefile.am

```diff
@@ -29,6 +29,10 @@ doc_DATA = TODO
 
 ACLOCAL_AMFLAGS = -I m4
 
+gsettings_SCHEMAS = org.autistici.gnome-mastermind.gschema.xml
```

Comment:
I'm not affiliated with autistici/inventati, it's just a network that provides hosting for hackers and free software passionates. I'd like to remove the "autistici" keyword from the schemas definition. Not sure what to replace it with though... maybe a super simple org.fargiolas.gnome-mastermind ?

Reply:
Sounds good to me!